.cache/v/cache/lastfailed

```diff
@@ -0,0 +1 @@
+{}
```

Comment:
Could you please add a line to the `.gitignore` file, so this file does not get commit:

```
# Unit test / coverage reports
.coverage
.tox
.cache   # <--- Add this line.
```